src/main/java/graphql/schema/SchemaTraverser.java

```diff
@@ -131,7 +133,10 @@ private static class TraverserDelegateListVisitor implements TraverserVisitor<Gr
         @Override
         public TraversalControl enter(TraverserContext<GraphQLSchemaElement> context) {
             for (GraphQLTypeVisitor graphQLTypeVisitor : typeVisitors) {
-                context.thisNode().accept(context, graphQLTypeVisitor);
+                TraversalControl control = context.thisNode().accept(context, graphQLTypeVisitor);
+                if (control != CONTINUE) {
+                    return control;
+                }
```

Comment:
This was broken.  I fixed it and now tests broke

So I wrote a new test for this case - when NOT CONTINUE is returned